### ISSUE TITLE
Wolcen: Lords of Mayhem - Rendering issues in main menu

### ISSUE THREAD

**Issue by mozo78** [doitsujin/dxvk#1115]
Hello,
In general the game working well for me but when I exit from active game to the main menu there are color and rendering issues which aren't present in the initial start - only when returning to main menu from active game:
![Screenshot_20190705_180708](https://user-images.githubusercontent.com/38666396/60732250-37929a00-9f52-11e9-9934-5e41ffc8641d.png)
See the Player name at the lower middle, the color rainbow in the upper left corner, etc. 
When I open settings and close them, I have this:
![Screenshot_20190705_180722](https://user-images.githubusercontent.com/38666396/60732595-25652b80-9f53-11e9-93fd-5529356b9eaa.png)
If I open them on initial start there isn't a problem and everything is working fine.

For reference, when I start the game for the first time and not played yet, the main menu looks like this:
![main menu](https://user-images.githubusercontent.com/38666396/60732480-cd2e2980-9f52-11e9-9036-4bd03a7dd9fc.png)
In the log I saw the following error which I think may be a part of the problem:

`warn:  D3D11Device: Vertex input 'COLOR0' not defined by input layout`

This problem isn't present with WineD3D.

Since the game has very long loading times I'm afraid the apitrace should be enormous but if I have to I'll try to porovide it.

### Software information
Wolcen: Lords of Mayhem

### System information
- GPU: GTX 1080 Ti
- Driver: 430.26
- Wine version: 4.11
- DXVK version:  1.2.3 and latest master

### Log files
- d3d11.log:
[Wolcen_d3d11.log](https://github.com/doitsujin/dxvk/files/3363318/Wolcen_d3d11.log)

- dxgi.log: 
[Wolcen_dxgi.log](https://github.com/doitsujin/dxvk/files/3363319/Wolcen_dxgi.log)

**Comment by doitsujin** [comment]
Apitrace?

**Comment by mozo78** [comment]
Here you are:
https://my.pcloud.com/publink/show?code=XZVR937Z7r0mVPDJ4Up99o4KABSxSfTH1267

**Comment by mozo78** [comment]
Unfortunately for this apitrace I have to install Windows for it's impossible to record it on Linux. I tested the game on Windows and I saw it runs with ~150 FPS. With DXVK it runs with ~70/75. Is there any obvious reason for such a big performance gap? Usually we expect 10-15% difference.

**Comment by doitsujin** [comment]
I don't have the game so I can't comment on that, especially with so little information given (are you CPU limited? GPU limited?)

**Comment by mozo78** [comment]
Neither CPU, neither VRAM/RAM:
![Screenshot_20190706_165901](https://user-images.githubusercontent.com/38666396/60757147-8baf8400-a00f-11e9-90e5-f0d3d67cdd7f.png)
![Screenshot_20190706_165919](https://user-images.githubusercontent.com/38666396/60757148-8f430b00-a00f-11e9-8d7c-7fed6e0d90a8.png)

**Comment by doitsujin** [comment]
46% GPU -> CPU limit. It's normal for the Windows Nvidia driver to be much faster in that case.

**Comment by mozo78** [comment]
I didn't understand you for english isn't my native language but if you say that's normal, ok :)

**Comment by aufkrawall** [comment]
The game might not batch drawcalls and then some secret Nvidia D3D11 driver sauce kicks in. At least that seemed to be the case in Kingdom Come: Deliv., there you could enable batching via the game's console and it boosted CPU performance a lot for the AMD driver.

**Comment by mozo78** [comment]
How do you do that?

Unfortunately DXVK 1.3 doesn't fix both the problems.

**Comment by mozo78** [comment]
Hmm, lowering the settings to their minimum doesn't affect the performance in any way.

**Comment by aufkrawall** [comment]
> How do you do that?

Found this: https://www.youtube.com/watch?v=GruK4j-Rr3c
The console key was disabled at some point in Wolcen, dunno how current situation is and if there is a way to get it working. Seems it would be crazy if this wasn't enabled by default.

**Comment by mozo78** [comment]
Thank you :)

**Comment by doitsujin** [comment]
To give you an update, the game uses counters for some inexplicable reason (they are primarily a debugging and profiling feature,  not something you should be using in a release build in your game...) which are not and cannot be supported by DXVK in any meaningful way. This is usually not an issue, but causes apitrace to crash so I'll have to figure out a workaround for that.

**Comment by mozo78** [comment]
Thank you very much for your kindness :)

**Comment by mozo78** [comment]
@doitsujin
Can I ask you something?
You say my CPU is the bottleneck for this game. How Intel 9900K can be the bottleneck and why?

**Comment by Joshua-Ashton** [comment]
@mozo78 The fact you have an Intel 9900K has nothing to do with it. The thing restricting you from getting better performance is you aren't able to submit work to the GPU fast enough (in layman's terms.)

**Comment by mozo78** [comment]
Thank you, Josh :)
Do you know what may be the reason?

**Comment by Joshua-Ashton** [comment]
Not off the top of my head, no.

**Comment by mozo78** [comment]
I see. Thank you.

**Comment by mozo78** [comment]
I tried the latest game version 1.1.5 with DXVK 1.3.2 but the problem is still here.

**Comment by doitsujin** [comment]
I worked around the counter problem with the apitrace, but I cannot reproduce the issue. There are also no Vulkan validation errors.

**Comment by mozo78** [comment]
No problem, thank you for your efforts :)

**Comment by lieff** [comment]
I have this issue sometimes, in menu and in game. Usually in game on first load when more disk activity. Just need restart game and issue goes away. Can be game bug or some synchronization issue.

**Comment by liberodark** [comment]
Not have bug in menu but have see on Nvidia Driver sometimes player is invisible or monsters.

**Comment by K0bin** [comment]
@liberodark Try updating your graphics driver first.

**Comment by doitsujin** [comment]
Problem apparently solved itself.
